interview-questions.md

```diff
@@ -8,32 +8,32 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What does it mean to implement a TDD workflow?
 
-Your answer:
+Your answer: Implementing TDD or Test driven development means that the test will be written first then code after to ensure that the test passes and that the test and code will be aligned.
 
-Researched answer:
+Researched answer: Test driven development  or TDD in the development world is when the developers write a test before they write just enough production code to fulfill the initially written test and subsequent refactoring is implemented after all has passed, developers also use the specifications and first write test describing how the code should behave.
 
 2. What is a relational database? Are there other kinds of databases?
 
-Your answer:
+Your answer: A relational database is a software or program used to create, read, update and destroy the specified database. I am unable to answer if there are other existing databases at this time and will be getting back to you with an answer.
 
-Researched answer:
+Researched answer: Relational database is a program used to create,update and manage relational databases.Some of the most well known databses include MySQL, PostgreSQL, MariaDB, MicrosoftSQL, and Oracle Databse to name a few.
```

Comment:
These answers are not correct.  Please help yourself out and find the correct answer. It's in the syllabus.  :)